controllers/configurationpolicy_controller_test.go

```diff
@@ -648,3 +650,383 @@ func TestShouldEvaluatePolicy(t *testing.T) {
 		)
 	}
 }
+
+func TestGetPolicyNamespaces(t *testing.T) {
+	t.Parallel()
```

Comment:
This can't be run in parallel because of the call to `Initialize` which affects global variables in the package.

Reply:
Ah, I didn't think of that. Thanks! Though I hadn't been running into issues (perhaps because it only creates namespaces and then retrieves them)? Regardless, probably safer to keep them isolated.